docs/topics/downloader-middleware.rst

```diff
@@ -745,6 +746,21 @@ Default: ``True``
 
 Whether the Compression middleware will be enabled.
 
+RETRY_HTTP_COMPRESSION_ERRORS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``False``
+
+Whether we should retry requests when an exception is raised while trying
```

Comment:
:lipstick: Style suggestions:
* I think in English “whether” only makes sense if there is an “or” somewhere. You can alternatively use “if” instead.
* I would avoid “we” here, and instead switch to passive.
* Since you use “its” in the next line, using singular “request” makes more sense; you could also change “its” to “theirs” instead, but I find that using singular in this context is more common in English.

```suggestion
Whether or not a request should be retried when an exception is raised while trying
```

Reply:
I'm used to native speakers omitting the _or not_ part of the _whether_ expression as it's most of the time redundant.

Here you can find some references:
- https://dictionary.cambridge.org/dictionary/english/whether
- https://english.stackexchange.com/questions/3382/whether-or-not-vs-whether

It's also being used this way by the `COMPRESSION_ENABLED` setting definition above.

In any case, I do find your suggested change better and I'm willing to commit it. Thank you.

Reply:
Thanks for the links! Good to know :smiley: